spec/rubocop/cop/rspec/expect_output_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::RSpec::ExpectOutput do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for overwriting $stdout' do
+    expect_violation(<<-RUBY)
+      specify do
+        $stdout = StringIO.new
+        ^^^^^^^ Use `expect { ... }.to output(...).to_stdout` instead of mutating $stdout
+      end
+    RUBY
+  end
+
+  it 'registers an offense for overwriting $stderr' do
```

Comment:
The description doesn’t properly describe the difference from the previous spec example. What’s special about `before(:each)` compared to `before(:suite)` in line 44?